back-nest/src/sсrapers/scraper-rozetka.service.ts

```diff
@@ -77,28 +77,28 @@ export class ScraperRozetkaService {
     }
   }
 
-  async clearRozetkaProducts(): Promise<void> {
-    try {
-      this.logger.log('Deleting products with source "ROZETKA"');
-      await this.productRepository.delete({ source: 'ROZETKA' });
-      this.logger.log('Successfully deleted products with source "ROZETKA"');
-    } catch (error) {
-      this.logger.error(
-        'Failed to delete products with source "ROZETKA"',
-        error,
-      );
-    }
-  }
-
-  async saveProductsToDB(products: Product[], source: string): Promise<void> {
-    if (!products.length) {
+  async saveProductsToDB(products: Product[]): Promise<void> {
+    if (!products || products.length === 0) {
```

Comment:
you can shorten it to next code:

```suggestion
    if (!products?.length) {
```